### PR TITLE
Update MobileFuse SDK to v1.6.5

### PR DESCRIPTION
**Pull request by tomaszbmf**
Use the latest MobileFuse SDK 1.6.5 in the Adapter.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.1.6.5.0
```
Comment (cb-jpadilla) [inline CHANGELOG.md:6]: There should be a README.md file that needs to be updated with this version as well.

**Review by cb-jpadilla — CHANGES_REQUESTED** [review]
Minor Comment. Needs update to the README.md

**Comment by tomaszbmf** [comment]
> Minor Comment. Needs update to the README.md

I missed it, thanks. I added the requested change.

**Review by cb-jpadilla — APPROVED** [review]
Thank you for the changes. Approved.

**Pull request merged**